install/kubernetes/helm/subcharts/grafana/templates/create-custom-resources-job.yaml

```diff
@@ -44,7 +44,7 @@ subjects:
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: istio-grafana-post-install
+  name: istio-grafana-post-install-v1.1.0
```

Comment:
why change the Job name here?

Reply:
This should be used to avoid upgrade collision.
It's hard to manage hooks during upgrade and helm is not complete on this part.

Reply:
Image is immutable in the job template spec. We need to create a new job (i.e. different name). Alternatively, we would need to delete the old job but our install process doesn't really allow for that.